api/base.proto

```diff
@@ -227,3 +227,7 @@ message TransportSocket {
   // See the supported transport socket implementations for further documentation.
   google.protobuf.Struct config = 2;
 }
+
+message Percent {
+  float value = 5 [(validate.rules).float = {gte: 0, lte: 100}];
```

Comment:
`float value = 1` would be more minimal. One other thing; I think @mattklein123 was suggesting we make this between 0.0 and 1.0.

Reply:
ah, sorry, I got lazy. Will fix.